README.release.md

```diff
@@ -0,0 +1,49 @@
+# Creating a New Release
+
+## Changelog
+
+The changelog is autogenerated by github at the release notes by inspecting 
+labels at PRs.
+
+To have a proper changelog, make sure the PRs are labeled as expected so they 
+they appear at proper the proper section according to [1] this are the labels:
+
+- Skip release notes:
+  - ignore-for-release
+- Breaking Changes 🛠
```

Comment:
Please note that while under the major version 0, breaking changes are allowed on the minor releases.